src/main/java/edu/hm/hafner/analysis/LineRangeList.java

```diff
@@ -98,7 +106,7 @@ public boolean contains(final Object o) {
         if (o instanceof LineRange) {
             LineRange lr = (LineRange) o;
 
-            for (Cursor c = new Cursor(); c.hasNext();) {
+            for (Cursor c = new Cursor(); c.hasNext(); ) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [';' is followed by whitespace.](https://app.codacy.com/manual/uhafner/analysis-model/pullRequest?prid=4887906)